### PR TITLE
Style/update about us page

### PR DESCRIPTION
**Pull request by YouQam**
This PR involves:
- updating `about-page` based on this design [here](https://www.figma.com/file/rq5IOcBWjVAl7tmvtq0GqS/Use-Cases?type=design&node-id=573-2160&mode=design&t=4ODpYZOoDEJ24hgE-0)
- updating navigation bar
- updating footer

_Note: This PR. doesn't support smart phone view, there's a separate card for it [here](https://github.com/orgs/mint-o-badges/projects/2/views/1?pane=issue&itemId=57078634)_

### CODE REVIEW

File: src/app/public/components/about/about.component.html [hunk 1]
```diff
@@ -1,84 +1,198 @@
...
-				{{ 'About.mainDesc' | translate }}
-			</p>
-			<p style="padding-top: 1.5rem">
-				{{ 'About.contact' | translate }}
-                <!--TODO: Update mail address here to open-educational-badges-->
-				<a href="mailto:info@opensenselab.org">info&#64;opensenselab.org </a>
-			</p>
+	<div style="position: absolute; right: 0; top: 11em">
+		<div>
+			<img class="badge-logo" src="../assets/logos/Badge-gelb.png" alt="Logo for badge gelb" />
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:14]: This alt text isn't very descriptive

File: src/app/public/components/about/about.component.html [hunk 2]
```diff
@@ -1,84 +1,198 @@
...
-					<div class="logo">
-						<a href="https://reedu.de/">
-							<img src="assets/logos/reedu.jpg" />
-						</a>
-					</div>
+			<div class="supporter-main-style">
+				<div>
+					<img
+						src="../assets/logos/Logo_Open-Sense-Lab.png"
+						alt="Logo for this website says: Coming soon: 01. April 2024"
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:33]: This alt text doesn't seem to fit here

File: src/app/public/components/about/about.component.html [hunk 3]
```diff
@@ -1,84 +1,198 @@
...
-					<div class="logo">
-						<a href="https://junge-tueftler.de/">
-							<img src="assets/logos/JT-Logo.png" />
-						</a>
-					</div>
+
+				<div>
+					<img
+						src="../assets/logos/Logo-Mycelia.png"
+						alt="Logo for this website says: Coming soon: 01. April 2024"
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:44]: This alt text doesn't seem to fit here

File: src/app/public/components/about/about.component.html [hunk 4]
```diff
@@ -1,84 +1,198 @@
...
-					<div class="logo">
-						<a href="https://ifgi.de/">
-							<img src="assets/logos/Logo_ifgi_long.png" />
-						</a>
-					</div>
+
+				<div>
+					<img
+						src="../assets/logos/Logo-Matrix.png"
+						alt="Logo for this website says: Coming soon: 01. April 2024"
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:55]: This alt text doesn't seem to fit here

File: src/app/public/components/about/about.component.html [hunk 5]
```diff
@@ -1,84 +1,198 @@
...
+						gGmbH, der mycelia gGmbH und der openSenseLab gGmbH als Verbund gemeinsam umgesetzt und an Mein
+						Bildungsraum angeschlossen. Gefördert wird das Projekt vom Bundesministerium für Bildung und
+						Forschung.
+					</p>
+				</div>
+			</div>
+
+			<div style="margin-right: 5em">
+				<div class="projects-logo-style">
+					<img src="../assets/logos/LogoKlimaCampus.png" alt="Logo for MintOBadges" />
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:134]: This alt text doesn't seem to fit here

File: src/app/public/components/about/about.component.html [hunk 6]
```diff
@@ -1,84 +1,198 @@
...
 			</div>
 		</div>
 	</div>
 
-	<div class="formsection formsection-nohelp u-margin-bottom4x">
-		<h2 class="formsection-x-title">{{ 'About.funding' | translate }}</h2>
+	<!-- Funded by -->
+	<div class="fund-style">
+		<div>
+			<img src="../assets/logos/Logo-Telekom-Stiftung.png" alt="Logo for BMBF" />
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:157]: This alt text doesn't seem to fit here

File: src/app/public/components/about/about.component.html [hunk 7]
```diff
@@ -1,84 +1,198 @@
...
-					</p>
-					<a
-						href="https://www.photonikforschung.de/projekte/open-innovation/foerdermassnahme/open-photonik-pro.html"
-						>{{ 'About.moreAboutFund' | translate }}</a
-					>
+		<div>
+			<img src="../assets/logos/Logo-BMBF.png" alt="Logo for BMBF" />
+		</div>
+
+		<div><img src="../assets/logos/Logo-BMUV.png" alt="Logo for BMBF" /></div>
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:164]: This alt text doesn't seem to fit here

**Review by timber-they — CHANGES_REQUESTED** [review]

**Comment by timber-they** [comment]
But is it *supposed* to not work on mobile?

File: src/app/public/components/about/about.component.html (hunk 7 above)
Comment (YouQam) [reply src/app/public/components/about/about.component.html:164]: Thanks timber for the review! This PR still in progress, I just created a pull request to send use the preview.

**Review by timber-they — APPROVED** [review]

**Pull request merged**